app/handlers/objectTracking.py

```diff
@@ -28,34 +32,56 @@ class ObjectTrackingHandler(tornado.web.RequestHandler):
 
     def post(self):
         # TODO: Implement rerun flag to prevent unnecessary computation
-        status_code, reason = self.handler(self.get_body_argument("identifier"))
+
         email = self.get_body_argument("email", default = None)
+        status_code, reason = self.handler(self.get_body_argument("identifier"), email)
+        
+        if status_code == 200:
+            self.finish("Object Tracking")
+        else:
+            raise tornado.web.HTTPError(reason=reason, status_code=status_code)
+
+    @staticmethod
+    def callback(status_code, response_message, email):
         if status_code == 200:
             if not email == None:
                 message = "Hello,\n\tWe have finished processing your video and identifying all objects.\nThank you for your patience,\nThe Santos Team"
                 subject = "Your video has finished processing."
-            
+
                 EmailHelper.send_email(email, subject, message)
-            self.finish("Object Tracking")
-        else:
-            raise tornado.web.HTTPError(reason=reason, status_code=status_code)
+
+        print (status_code, response_message)
 
     @staticmethod
-    def handler(identifier):
+    def handler(identifier, email):
         """
         Runs TrafficIntelligence trackers and support scripts.
         """
         project_path = get_project_path(identifier)
         if not os.path.exists(project_path):
            return (500, 'Project directory does not exist. Check your identifier?')
 
+        ObjectTrackingThread(identifier, email, ObjectTrackingHandler.callback).start()
+
+        return (200, "Success")
+
+class ObjectTrackingThread(threading.Thread):
```

Comment:
We should eventually convert this to using the multiprocessing library (threadpools and such)